docs/pages/includes/teleport-connect-telemetry.mdx

```diff
@@ -0,0 +1,18 @@
+When you first start the app, Teleport Connect asks for permission to collect & send telemetry data.
+This includes tracking events such as:
+- Login
+- Starting an SSH, database, or kube session
+- File transfer during an SSH session
+- Managing access requests
+- Asking about job role
```

Comment:
Any suggestions? 
We ask users about it to get a list of job roles for people who are using Teleport Connect. Nothing better than "job role" comes to my mind 😏

Reply:
Ah, that make sense. I wasn't sure what context to use for interpreting "job role". Maybe something like, "Indicating when Teleport Connect asks a user to describe their role at their organization"?

Are all the items in this list events that Teleport Connect tracks? If the job title item isn't, I think we should separate it from this list.

Reply:
Ah, I realized that including "job role" it the list is confusing, because the event is not intended to indicate that the user was asked, we collect the user's response itself.
What do you think of adding a line below `- Processor architecture`:
```
Additionally, we ask for a job role (answer is optional).
```

Reply:
That sounds good to me!

Reply:
Done